waspc/src/Wasp/Generator/Common.hs

```diff
@@ -14,12 +15,22 @@ data ProjectRootDir
 -- | Node version that node packages generated by this generator expect.
 --   (major, minor, patch)
 nodeVersion :: (Int, Int, Int)
-nodeVersion = (12, 18, 0) -- Latest LTS version.
+nodeVersion = (16, 0, 0) -- Latest LTS version.
 
 nodeVersionAsText :: String
-nodeVersionAsText = printf "%d.%d.%d" major minor patch
-  where
-    (major, minor, patch) = nodeVersion
+nodeVersionAsText = versionAsText nodeVersion
 
-prismaVersion :: String
-prismaVersion = "3.9.1"
+npmVersion :: (Int, Int, Int)
+npmVersion = (8, 0, 0) -- Latest LTS version.
+
+npmVersionAsText :: String
+npmVersionAsText = versionAsText npmVersion
+
+prismaVersion :: (Int, Int, Int)
+prismaVersion = (3, 9, 1) -- Latest LTS version.
+
+prismaVersionAsText :: String
+prismaVersionAsText = versionAsText prismaVersion
```

Comment:
I wanted to make this consistent with the others:
- `fooVersion` - tuple of Ints
- `fooVersionAsString` - text